src/core/cgroup.c

```diff
@@ -4573,6 +4573,7 @@ static uint64_t unit_get_effective_limit_one(Unit *u, CGroupLimitType type) {
                 }
 
         cc = unit_get_cgroup_context(u);
+        assert(cc);
```

Comment:
assert_se?

Reply:
It's not a test code and the expression has no side effects, so regular `assert()` is appropriate here.

Reply:
(I meant to merge with the line above)

Reply:
Nitpickers, assemble!